src/Covid19Dashboard/index.jsx

```diff
@@ -255,6 +255,7 @@ class Covid19Dashboard extends React.Component {
               <Tab>COVID-19 in Illinois</Tab>
               <Tab>COVID-19 in the world</Tab>
               <Tab>SARS-CoV-2 Genomics</Tab>
+              <Tab>SARS-CoV-2 Genomics IL</Tab>
```

Comment:
Gina asked to change the tab names to `Global SARS-CoV2 Genomics` and `IL SARS-CoV2 Genomics`

Reply:
probably need to change the iframe titles as well

Reply:
I changed those